uber/templates/preregistration/duplicate.html

```diff
@@ -3,7 +3,8 @@
 {% block backlink %}{% endblock %}
 {% block content %}
 
-    <div class="panel panel-default">
+<div class="masthead"></div>
+<div class="panel panel-default">
```

Comment:
need to close this div.  (sidenote: not sure why this is part of this PR, though it may have been a change pulled from master just being displayed by github)